.github/workflows/docsite-publish.yml

```
@@ -0,0 +1,33 @@
+# Workflow name
+name: 'Docsite publish'
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.18.1
+
+      - name: Install packages
+        run: ./yarn-ci.sh
+        shell: bash
+
+      - name: Build dependencies
+        run: yarn build --to @fluentui/react --to @fluentui/react-components --to @fluentui/storybook
```

```suggestion
        run: yarn build --to @fluentui/public-docsite-v9
```

why not just build to the docsite project ?

---

good catch

---

Sounds good! Let me double check this on local.

---

All good, change commited and pushed.